usaspending_api/etl/management/commands/load_subawards.py

```diff
@@ -60,17 +62,29 @@ def handle(self, *args, **options):
         else:
             db_cursor = PhonyCursor()
 
-        submission_ids_to_update = []
+        submissions_to_update = []
 
         if options["update_all"]:
             submissions_to_update = SubmissionAttributes.objects.exclude(broker_submission_id__isnull=True)
         else:
-            submission = SubmissionAttributes.objects.filter(broker_submission_id=options['submission_id']).first()
-            if not submission:
-                logger.critical("Submission {} not found in datastore".format(options['submission_id']))
-                return
-            submissions_to_update = [submission]
+            for submission_id in options['submission_id']:
+                sub = SubmissionAttributes.objects.filter(broker_submission_id=submission_id).first()
+                if not sub:
+                    logger.critical("Submissions not found in datastore".format(options['submission_id']))
+                else:
+                    submissions_to_update.append(sub)
 
+        failed_submissions = []
+        success_submissions = []
         for submission in submissions_to_update:
-            logger.info("Loading subaward data for submission {}".format(submission.broker_submission_id))
-            load_subawards(submission, db_cursor)
+            try:
```

Comment:
I'm torn here...I see the need for a soft landing as we're loading all of this data, but am also wary of wrapping everything in a `try/catch` with a generic exception. Are there more specific exceptions we can anticipate and handle?